toolchain-m68k

```diff
@@ -241,6 +241,14 @@ def build():
                               path.join('{host}', 'bin'),
                               environ['PATH']])
 
+  """
+  When we have a working compiler in our path, we can also check if the
+  required headers/libraries are present.
+  """
+
+  require_header('<python2.7/python.h>')
```

Comment:
You're already doing exactly that:
````
if not sys.version_info[:2] == (2, 7):
  panic('I need Python 2.7 to run!')
```
The prerequisites say "python-dev", wouldn't the presence of python.h indicate that all of those things are installed, and not just the run-time files?